ci/tests/api-functionality/docker-compose.yml

```diff
@@ -16,7 +22,28 @@ services:
       - TYK_DB_REDISHOST=redis
       - TYK_LOGLEVEL=debug
 
+  gw-checker:
+    image: curlimages/curl
+    networks: [ test-api-functionality ]
+    command: tail -f /dev/null
+    depends_on:
+      - gw
+    healthcheck:
+      test: curl -s --fail http://gw:8080/hello | grep -o '.*{"redis":{"status":"pass",'
```

Comment:
The check with grep could fail based on order of keys in the json objects. Asserting the value with jq will be more reliable but the base image needs to provide jq.

This image is ~5mb and provides curl and jq: https://hub.docker.com/r/badouralix/curl-jq

Reply:
https://tyktech.atlassian.net/browse/TD-2072